plugins/regex_remap/regex_remap.cc

```diff
@@ -461,8 +475,9 @@ RemapRegex::compile(const char **error, int *erroffset)
 
       if (ix > -1) {
         if ((ix < 10) && (ix > ccount)) {
-          TSDebug(PLUGIN_NAME, "Trying to use unavailable substitution, check the regex!");
-          return -1; // No substitutions available other than $0
+          error = "using unavailable captured substring ($n) in substitution";
+          TSDebug(PLUGIN_NAME, error);
```

Comment:
True, it's redundant.  It could be:

TSDebug(PLUGIN_NAME, "using $%d in substitution string, $%d is highest available", ix, ccount);

or just deleted.